internal/commands/push/command.go

```diff
@@ -198,7 +198,35 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
 			return err
 		}
-		ui.Print(terminal.NewTextLog("Uploaded dependencies archive"))
+
+		ui.Print(terminal.NewTextLog("Uploaded dependencies"))
```

Comment:
Is this message necessary? If so, I thought it would be good to make it more consistent with the "Installed dependencies" message that should occur after it.

Reply:
It does seem like it can go away now... it previously existed to mark the completion of this block, which as you allude to is now done with the "Installed dependencies" message below

I think the key here is we want to make sure this behaves similarly (read: prints the same amount of logs) as the hosting and/or reset cdn blocks do below (which from the looks of it appears to be the case)

Have you tested this locally yet?  I'm guessing there's a test case below that can also confirm this...

Reply:
Yes, as you saw below in the tests it ends up being 2 dependencies-related messages in the tests so I will remove it.